### PR TITLE
[RDAPI-29348] Update Aug23 release notes for Java 11 update

### PR DESCRIPTION
**Pull request by JohnD-Axway**
http://pr-2709.axway-open-docs.opendocs-builder.pcloud.axway.int/docs/apim_relnotes/20230830_apimgr_relnotes/#java-version-update

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | 45bc386 |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-2709.axway-open-docs.opendocs-builder.pcloud.axway.int |

File: content/en/docs/apim_relnotes/20230830_apimgr_relnotes.md [hunk 1]
```diff
@@ -40,6 +40,39 @@ In our Helm chart, it is now possible to add custom initContainers to any of the
...
+
+1. **Java Version Update:**
+
+The application has been upgraded from Java 8 to Java 11. The specific version is **Azul Zulu11.64+19-CA** (build on OpenJDK **11.0.19+7-LTS**).  With this update certain Java 8 features and APIs that are now deprecated or removed might affect your custom code, scripts, and integrations.
+
+2. **Deprecated and Removed APIs:**
+
+Java 11 has deprecated and removed several APIs that were present in Java 8. Please review your custom Java code, JS scripts, and Groovy scripts to identify and update any references to these deprecated or removed APIs. The most common impacted areas include:
+
+ - `javax.xml.bind package`: This package has been removed. If your code relies on JAXB for XML binding, you'll need to migrate to alternative libraries such as jakartat.xml.bind.
```
Comment (Bcolemanaxway) [inline content/en/docs/apim_relnotes/20230830_apimgr_relnotes.md:55]: Typo on jakarta.xml.bind

**Comment by jolearyaxway** [comment]
Is there a note elsewhere about the keystore passphrase required to match the key pass phrase or was that resolved in another way?
Is it worth noting the switch to PKCS12 by default now for new keystores?
Any note needed about jdbc11 ?

**Review by jolearyaxway — COMMENTED** [review]
Is there a note elsewhere about the keystore passphrase required to match the key pass phrase or was that resolved in another way?
Is it worth noting the switch to PKCS12 by default now for new keystores?
Any note needed about jdbc11 ?

File: content/en/docs/apim_relnotes/20230830_apimgr_relnotes.md (hunk 1 above)
Comment (JohnD-Axway) [reply content/en/docs/apim_relnotes/20230830_apimgr_relnotes.md:55]: good catch

**Review by mbarova — APPROVED** [review]

**Review by Bcolemanaxway — CHANGES_REQUESTED** [review]

File: content/en/docs/apim_relnotes/20230830_apimgr_relnotes.md (hunk 1 above)
Comment (JohnD-Axway) [reply content/en/docs/apim_relnotes/20230830_apimgr_relnotes.md]: This and other comments adressed!

**Review by Bcolemanaxway — APPROVED** [review]

File: content/en/docs/apim_relnotes/20230830_apimgr_relnotes.md [hunk 2]
```diff
@@ -52,10 +52,10 @@ API Gateway now runs on Java 11. This upgrade will allow the development of new
...
+ * `javax.xml.bind package`: This package has been removed. If your code relies on JAXB for XML binding, you'll need to migrate to alternative libraries such as jakarta.xml.bind.
+ * java.security APIs: Certain cryptographic algorithms and security protocols might be deprecated or removed. Ensure that your security configurations are up-to-date. Older certificates, for example, may no longer be compatible with the default java.security settings.
+ * OpenJSSE: OpenJSSE is not required in Java 11 and is no longer shipped with the gateway. If customizations have direct dependencies on OpenJSSE they will need to be updated.
+ * Function changes: Some functions may have subtle changes to their output, this is not always covered by documentation. When tracking erroneous behavior all function outputs should be checked. For example the function `InetAddress.getLocalHost().getHostAddress()` no longer returns the fully qualified domain name (FQDN) and the method `getCanonicalHostName()`
```
Comment (scaro-axway) [inline content/en/docs/apim_relnotes/20230830_apimgr_relnotes.md:58]: It is '.getHostName()' that is replaced by '.getCanonicalHostName()', not the '.getHostAddress()'.

Comment (andreamussap) [reply content/en/docs/apim_relnotes/20230830_apimgr_relnotes.md:58]: Fixed, thanks!

**Pull request merged**